### PR TITLE
Expose inverse relationships on api

### PR DESCRIPTION
**Pull request by mmadariaga**
Expose features on platform /brand endpoints (GET/{id}, POST and PUT)

#### Type Of Change 
- [x] New feature or enhancement

#### Checklist:
- [X] Commits are named and have tag following [commit rules](https://github.com/irontec/ivozprovider/blob/bleeding/doc/dev/en/commits.md)
- [X] Commits are split per component (schema, web/admin, kamusers, agis, ..)
- [X] Changes have been tested locally

### CODE REVIEW

File: library/Ivoz/Provider/Application/Service/ExternalCallFilter/ExternalCallFilterDtoAssembler.php [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+use Ivoz\Core\Application\Service\Assembler\CustomDtoAssemblerInterface;
+use Ivoz\Provider\Domain\Model\ExternalCallFilter\ExternalCallFilterDto;
+use Ivoz\Provider\Domain\Model\ExternalCallFilter\ExternalCallFilterInterface;
+use Assert\Assertion;
+use Ivoz\Provider\Domain\Model\ExternalCallFilterBlackList\ExternalCallFilterBlackList;
+use Ivoz\Provider\Domain\Model\ExternalCallFilterRelCalendar\ExternalCallFilterRelCalendar;
+use Ivoz\Provider\Domain\Model\ExternalCallFilterRelSchedule\ExternalCallFilterRelSchedule;
+use Ivoz\Provider\Domain\Model\ExternalCallFilterWhiteList\ExternalCallFilterWhiteList;
+use Ivoz\Provider\Domain\Model\Feature\Feature;
+use Ivoz\Provider\Domain\Model\PickUpRelUser\PickUpRelUser;
```
Comment (Kaian) [inline library/Ivoz/Provider/Application/Service/ExternalCallFilter/ExternalCallFilterDtoAssembler.php]: Arte these two `use` required?

use Ivoz\Provider\Domain\Model\Feature\Feature;
use Ivoz\Provider\Domain\Model\PickUpRelUser\PickUpRelUser;

File: library/Ivoz/Provider/Application/Service/ConditionalRoutesCondition/ConditionalRoutesConditionDtoAssembler.php [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+use Ivoz\Core\Application\Service\Assembler\CustomDtoAssemblerInterface;
+use Ivoz\Provider\Domain\Model\ConditionalRoutesCondition\ConditionalRoutesConditionDto;
+use Ivoz\Provider\Domain\Model\ConditionalRoutesCondition\ConditionalRoutesConditionInterface;
+use Assert\Assertion;
+use Ivoz\Provider\Domain\Model\ConditionalRoutesConditionsRelCalendar\ConditionalRoutesConditionsRelCalendar;
+use Ivoz\Provider\Domain\Model\ConditionalRoutesConditionsRelMatchlist\ConditionalRoutesConditionsRelMatchlist;
+use Ivoz\Provider\Domain\Model\ConditionalRoutesConditionsRelRouteLock\ConditionalRoutesConditionsRelRouteLock;
+use Ivoz\Provider\Domain\Model\ConditionalRoutesConditionsRelSchedule\ConditionalRoutesConditionsRelSchedule;
+use Ivoz\Provider\Domain\Model\Feature\Feature;
+use Ivoz\Provider\Domain\Model\PickUpRelUser\PickUpRelUser;
```
Comment (Kaian) [inline library/Ivoz/Provider/Application/Service/ConditionalRoutesCondition/ConditionalRoutesConditionDtoAssembler.php]: Review `use` directives

File: library/Ivoz/Provider/Application/Service/Company/CompanyDtoAssembler.php [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+use Ivoz\Core\Application\DataTransferObjectInterface;
+use Ivoz\Core\Application\Service\StoragePathResolverCollection;
+use Ivoz\Core\Domain\Model\EntityInterface;
+use Ivoz\Core\Application\Service\Assembler\CustomDtoAssemblerInterface;
+use Ivoz\Provider\Domain\Model\Company\CompanyDto;
+use Ivoz\Provider\Domain\Model\Company\CompanyInterface;
+use Assert\Assertion;
+use Ivoz\Provider\Domain\Model\Feature\Feature;
+use Ivoz\Provider\Domain\Model\FeaturesRelCompany\FeaturesRelCompany;
+use Ivoz\Provider\Domain\Model\FeaturesRelCompany\FeaturesRelCompanyInterface;
```
Comment (Kaian) [inline library/Ivoz/Provider/Application/Service/Company/CompanyDtoAssembler.php]: Review `use` directives

File: library/Ivoz/Provider/Application/Service/Ivr/IvrDtoAssembler.php [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+use Ivoz\Core\Application\DataTransferObjectInterface;
+use Ivoz\Core\Application\Service\StoragePathResolverCollection;
+use Ivoz\Core\Domain\Model\EntityInterface;
+use Ivoz\Core\Application\Service\Assembler\CustomDtoAssemblerInterface;
+use Ivoz\Provider\Domain\Model\Ivr\IvrDto;
+use Ivoz\Provider\Domain\Model\Ivr\IvrInterface;
+use Assert\Assertion;
+use Ivoz\Provider\Domain\Model\Feature\Feature;
+use Ivoz\Provider\Domain\Model\IvrExcludedExtension\IvrExcludedExtension;
```
Comment (Kaian) [inline library/Ivoz/Provider/Application/Service/Ivr/IvrDtoAssembler.php]: Review `use` directives

File: library/Ivoz/Provider/Application/Service/OutgoingRouting/OutgoingRoutingDtoAssembler.php [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+use Ivoz\Core\Application\DataTransferObjectInterface;
+use Ivoz\Core\Application\Service\StoragePathResolverCollection;
+use Ivoz\Core\Domain\Model\EntityInterface;
+use Ivoz\Core\Application\Service\Assembler\CustomDtoAssemblerInterface;
+use Ivoz\Provider\Domain\Model\OutgoingRouting\OutgoingRoutingDto;
+use Ivoz\Provider\Domain\Model\OutgoingRouting\OutgoingRoutingInterface;
+use Assert\Assertion;
+use Ivoz\Provider\Domain\Model\Feature\Feature;
+use Ivoz\Provider\Domain\Model\OutgoingRoutingRelCarrier\OutgoingRoutingRelCarrier;
```
Comment (Kaian) [inline library/Ivoz/Provider/Application/Service/OutgoingRouting/OutgoingRoutingDtoAssembler.php]: Review `use` directives

File: library/Ivoz/Provider/Application/Service/PickUpGroup/PickUpGroupDtoAssembler.php [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+namespace Ivoz\Provider\Application\Service\PickUpGroup;
+
+use Ivoz\Core\Application\DataTransferObjectInterface;
+use Ivoz\Core\Application\Service\StoragePathResolverCollection;
+use Ivoz\Core\Domain\Model\EntityInterface;
+use Ivoz\Core\Application\Service\Assembler\CustomDtoAssemblerInterface;
+use Ivoz\Provider\Domain\Model\PickUpGroup\PickUpGroupDto;
+use Ivoz\Provider\Domain\Model\PickUpGroup\PickUpGroupInterface;
+use Assert\Assertion;
+use Ivoz\Provider\Domain\Model\Feature\Feature;
```
Comment (Kaian) [inline library/Ivoz/Provider/Application/Service/PickUpGroup/PickUpGroupDtoAssembler.php]: Review `use` directives

File: library/Ivoz/Provider/Application/Service/User/UserAssembler.php [hunk 7]
```diff
@@ -30,6 +30,9 @@ public function fromDto(
         }
         // There is not oldPass validation in klear, so, we can't do any further validation
 
+
+
+
```
Comment (Kaian) [inline library/Ivoz/Provider/Application/Service/User/UserAssembler.php]: The more space the merrier.

File: library/Ivoz/Provider/Application/Service/User/UserDtoAssembler.php [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+
+use Ivoz\Core\Application\DataTransferObjectInterface;
+use Ivoz\Core\Application\Service\StoragePathResolverCollection;
+use Ivoz\Core\Domain\Model\EntityInterface;
+use Ivoz\Core\Application\Service\Assembler\CustomDtoAssemblerInterface;
+use Ivoz\Provider\Domain\Model\PickUpRelUser\PickUpRelUser;
+use Ivoz\Provider\Domain\Model\User\UserDto;
+use Ivoz\Provider\Domain\Model\User\UserInterface;
+use Assert\Assertion;
+use Ivoz\Provider\Domain\Model\Feature\Feature;
```
Comment (Kaian) [inline library/Ivoz/Provider/Application/Service/User/UserDtoAssembler.php]: Review `use` directives

**Comment by ironArt3mis** [comment]
Rebase completed.

**Comment by ironArt3mis** [comment]
Rebase completed.

**Pull request merged**